drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -64,17 +64,6 @@
 #define STM32WL_DUAL_CORE
 #endif
 
-#if STM32_AHB_PRESCALER > 1
```

Comment:
The same comment is present in stm32U5 specific file and should be removed

Reply:
removed

Reply:
It is no more written

Reply:
It is still present:
https://github.com/zephyrproject-rtos/zephyr/blob/ed1d1966e7f25057f4a8a494fe35512d56eafd88/drivers/clock_control/clock_stm32_ll_u5.c#L34